setup.cfg

```diff
@@ -0,0 +1,15 @@
+[aliases]
+test=pytest
```

Comment:
You must have copied this from one of the newer projects. This is the reason for the test failures. Not that using pytest on this project would be a bad thing, but one step at a time. 😀